### PR TITLE
Brushed up the source code

### PR DESCRIPTION
**Pull request by Helveg**
Maintainer can now `python setup.py bdist_wheel` to make a functional wheel.

### CODE REVIEW

**Comment by jougs** [comment]
Thanks for this contribution!

Can you please quickly comment in how far this complements of conflicts with #1? At least the move of files seems to be incompatible.

**Comment by Helveg** [comment]
It seems that #1 places the files in the root folder, I wouldn't do that. It's more conventional for Python projects to gather their source code in a folder named after the package (i.e. `nest_connplotter` if the package name is to be `nest-connplotter` as in #1). This will simplify all packaging and other convention-sensitive operations

**Comment by jougs** [comment]
I commented the same thing in #1 (i.e., files in sub-directories). Does this PR then replace #1?

**Comment by Helveg** [comment]
@jougs I added the missing changes from that PR here and now it can replace it

**Review by pnbabu — COMMENTED** [review]
@Helveg thanks for these changes! Isn't it better to have the package name as `nest-connplotter` instead of `ConnPlotter`?

**Comment by Helveg** [comment]
Both names are available on PyPI. Using `nest-connplotter` would mean that we have a different module name than package name with `nest_connplotter` as its closest match. I don't know if anyone else has any suggestions? `nestplotlib`? 😂 I believe it is PEP8 that recommends module names to be underscore cased, but it's in one of the PEPs for sure. I left it at `ConnPlotter` as I was impartial and wanted the least amount of refactoring work ;)

**Comment by jougs** [comment]
I have no strong opinions either way, but would like things to be somewhat consistent. Here's the current situation:

| Repository | PyPI package | Module |
|---|---|---|
| nest-simulator | nest-simulator | nest |
| ode-toolbox | odetoolbox |  odetoolbox |
| nestml | nestml | nestml |
| nest-sionlib-reader | | nestio |
| nest-connplotter | | |
| nest-desktop* | *nest-desktop?* | |
| nest-client* | *nest-client?* | |

\* soon in a NEST Initiative GitHub account near you.

@heplesser, @terhorstd: maybe we should discuss this in the next open VC?

**Comment by jougs** [comment]
@Helveg: would you please be so kind and amend 6de4e39 to add @pnbabu as co-author? Thanks!

**Comment by Helveg** [comment]
I can amend the commit message, is there a format I should follow, could you perhaps give me the exact string to amend?

**Comment by jougs** [comment]
Adding `Co-authored-by: Pooja Babu <p.babu@fz.juelich.de>` on a separate line should [do the trick](https://docs.github.com/en/github/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors).

**Comment by heplesser** [comment]
@jougs @terhorstd Putting the naming onto the agenda of the nest Open VC is a good idea.

**Comment by terhorstd** [comment]
:+1: Added as discussion point to [the next meeting](https://github.com/nest/nest-simulator/wiki/2021-09-13-Open-NEST-Developer-Video-Conference).

**Comment by jougs** [comment]
The naming discussion in the [Open NEST video conference today](https://github.com/nest/nest-simulator/wiki/2021-09-13-Open-NEST-Developer-Video-Conference) resulted in the creation of https://github.com/nest/nest-simulator/issues/2161 and the decision to rename this repository from `nest-connplotter` to `connplotter`. All further naming and consistency discussion should continue there.

These steps should now allow to also name the PyPI package and the module the same way as the repository. @Helveg: could you please update this PR accordingly? Thanks!

**Review by pnbabu — APPROVED** [review]
Looks good to me. Thanks!

File: setup.py [hunk 1]
```diff
@@ -21,16 +21,16 @@
...
 # ConnPlotter --- A Tool to Generate Connectivity Pattern Matrices
 
-from distutils.core import setup
+from setuptools import setup
 
-setup(name='ConnPlotter',
+setup(name='connplotter',
       version='0.7a0',
       description=('ConnPlotter is a tool to create ' +
                    'connectivity pattern tables'),
```
Comment (jougs) [inline setup.py]:
```suggestion
      description=('connplotter is a tool to create connectivity pattern tables'),
```

**Review by jougs — APPROVED** [review]
Just a minor suggestion.

**Comment by jougs** [comment]
When merging this one, https://github.com/nest/nest-simulator/pull/2147 should also be merged.

**Pull request merged**